scalgrafanalib/core.py

```diff
@@ -50,18 +51,37 @@ def to_json_data(self) -> Json:
 
 
 @attr.s
-class Stat(core.Stat):
+class Stat(core.Stat):  # pylint: disable=too-many-instance-attributes
     """Stat: Allow settings minValue and maxValue"""
 
     minValue = attr.ib(default=None)  # pylint: disable=invalid-name
     maxValue = attr.ib(default=None)  # pylint: disable=invalid-name
+    # Advanced options for version panel and others
+    reduceOptions = attr.ib(default=None)  # pylint: disable=invalid-name
```

Comment:
there is already `reduceCalc` and `fields` in Stat : do we need more (i.e. specify `values`) ?

```
                    'reduceOptions': {
                        'calcs': [
                            self.reduceCalc
                        ],
                        'fields': self.fields,
                        'values': False
                    }
```

Instead of `reduceOptions`, should we not just allow specifying/overriding the `values` field?